### ISSUE TITLE
Query log: missing label for input field

### ISSUE THREAD

**Issue by reynir** [robur-coop/dnsvizor#82]
> All interactive elements should have a name or label, otherwise a screen reader will not say what it is for.

<img width="214" height="50" alt="Image" src="https://github.com/user-attachments/assets/3e0215f3-08cf-4976-a094-b138321118f9" />

<img width="145" height="39" alt="Image" src="https://github.com/user-attachments/assets/3fd5292e-3a2c-4ac6-bb87-b678f50594e0" />

[4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
